web/client-api/src/main/java/io/deephaven/web/client/api/parse/JsDataHandler.java

```diff
@@ -137,7 +137,7 @@ public void write(Object[] data, ParseContext context, JsConsumer<Node> addNode,
             addNode.apply(new Node(data.length, nullCount));
         }
     },
-    DATE_TIME(Type.Int, "datetime", "java.time.Instant", "java.time.ZonedDateTime") {
+    DATE_TIME(Type.Int, "java.time.Instant", "datetime", "java.time.ZonedDateTime") {
```

Comment:
Should we also bump `ZonedDateTime` up?
Change seems innocuous. What's the effect, generally?

Reply:
See the constructor body - the first element in the string array is used as "what should we define as `deephaven:type` for this column", and all elements are used as "accept this from the js client user as a string to represent this type"

```java
    JsDataHandler(int arrowTypeType, String... typeNames) {
        this.arrowTypeType = arrowTypeType;
        assert typeNames.length > 0 : "Must have at least one name";
        this.deephavenType = typeNames[0];
        for (int i = 0; i < typeNames.length; i++) {
            JsDataHandler existing = HandlersHolder.HANDLERS.put(typeNames[i], this);
            assert existing == null : "Handler already registered for type " + typeNames[i] + ": " + name();
        }
    }
```